gateway/apiversion/apiversion.go

```diff
@@ -0,0 +1,3 @@
+package apiversion
+
+const LatestApiVersion uint64 = 20240214
```

Comment:
Is there a reason not to have the type alias for `ApiVersion` as on master?

Reply:
It's just backporting the original change, I don't expect this number should change at all.